keyboards/et/et7233/keyboard.json

```diff
@@ -0,0 +1,218 @@
+{
+    "manufacturer": "HS",
+    "keyboard_name": "ET 7233",
+    "maintainer": "yangzheng20003",
+    "bootloader": "wb32-dfu",
+    "diode_direction": "ROW2COL",
+    "eeprom": {
+        "driver": "wear_leveling",
```

Comment:
Default behaviour.
```suggestion
```

Reply:
Thank you for reminding me, I have revised it.